dlp/snippets/src/test/java/dlp/snippets/JobsTests.java

```diff
@@ -125,4 +134,19 @@ public void testDeleteJobs() throws Exception {
     String output = bout.toString();
     assertThat(output).contains("Job deleted successfully.");
   }
+
+  @Test
+  public void testCreateJobsSendScc() throws Exception {
+    // Call createJobs to create a Dlp job from project id and gcs path and send data to SCC.
+    InspectGcsFileSendToScc.createJobSendToScc(PROJECT_ID, GCS_PATH);
+    String output = bout.toString();
+    assertThat(output).contains("Job created successfully:");
+
+    // Delete the created Dlp Job
+    String dlpJobName = output.split("Job created successfully: ")[1].split("\n")[0];
```

Comment:
Please clarify why are we plucking out [1] and [0] here.

Reply:
I have modified the extraction here, we don't need [0] here.
So now .split("Job created successfully: ") splits the output string into an array of substrings using the string "Job created successfully: " as the delimiter. The result is an array with at least two elements. [1] accesses the second element (index 1) of the resulting array.

Reply:
@jakubrauch Since I have used `System.out.println` [here](https://github.com/GoogleCloudPlatform/java-docs-samples/pull/7922/files#diff-59a0bdcd72ddfe723b474c967233c2b82cdda57a211e7a01b1fb6f90d893c29eR112) , `.split("Job created successfully: ") ` splits the output string using the delimiter "Job created successfully: ", which returns an array of strings with at least two elements. The second element of the resulting array contains the job name along with the newline character.

The second split() method is used to remove the newline character from the job name. It splits the second element of the resulting array using the newline character (\n) as the delimiter, and accesses the first element of the resulting array, which should be the job name itself.